### PR TITLE
Upgrade to sbt-conductr 2.1.13

### PR DESCRIPTION
**Pull request by fsat**
- Set CPU requirement to `2.0`.
- Ensure configuration bundle can be loaded with provided example

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -17,12 +17,12 @@ lazy val root = (project in file(".")).
...
     BundleKeys.system := "postgres94",
     BundleKeys.bundleType := Docker,
-    BundleKeys.nrOfCpus := 4.0,
+    BundleKeys.nrOfCpus := 2.0,
     BundleKeys.memory := 2.GB,
     BundleKeys.diskSpace := 10.GB,
     BundleKeys.roles := Set("postgres94"),
     BundleKeys.endpoints := Map(
-      "postgres" -> Endpoint("tcp", 5432, services = Set(uri("tcp://:5432")))
+      "postgres" -> Endpoint("tcp", 5432, "postgres", RequestAcl(Tcp(5432)))
```
Comment (huntc) [inline build.sbt]: This will mean no more compatibility with 1.1. I think we should retain services here.

**Review by huntc — CHANGES_REQUESTED** [review]

File: build.sbt (hunk 1 above)
Comment (fsat) [reply build.sbt]: Oh yes - you are quite right. I'll revert back to services.

Comment (fsat) [reply build.sbt]: This is now reverted back to services.

**Comment by fsat** [comment]
Manual successfully test done on the AWS cluster.

The ConductR version tested requires the version with this change included:

https://github.com/typesafehub/conductr/pull/1491

The Postgrest BDR bundle and the example config is built, loaded, and scaled to 3 instances.

```
ubuntu@ip-10-0-3-168:~$ conduct info
ID               NAME                    #REP  #STR  #RUN
02e7452-b564a24  conductr-elasticsearch     3     0     3
9c722e7          conductr-haproxy           3     0     3
eb1df49-8d3dfaa  postgres-bdr               3     0     3
```

The logs also indicates successful startup.

```
ubuntu@ip-10-0-3-168:~$ conduct logs eb1df49-8d3dfaa
TIME      HOST           LOG
05:31:47  ip-10-0-1-45   LOG:  starting logical decoding for slot "bdr_12156_6347497381196554260_1_12156__"
DETAIL:  streaming transactions committing after 0/187F5F8, reading WAL from 0/187F590
05:31:47  ip-10-0-3-168  LOG:  starting logical decoding for slot "bdr_12156_6347497381196554260_1_12156__"
DETAIL:  streaming transactions committing after 0/1862FF0, reading WAL from 0/1862FB8
LOG:  logical decoding found consistent point at 0/1862FB8
DETAIL:  There are no running transactions.
05:31:47  ip-10-0-2-127  LOG:  logical decoding found consistent point at 0/18631B0
DETAIL:  There are no running transactions.
05:31:47  ip-10-0-2-127  LOG:  exported logical decoding snapshot: "000002C5-1" with 0 transaction IDs
05:31:47  ip-10-0-2-127  LOG:  starting logical decoding for slot "bdr_12156_6347497337961975827_1_12156__"
DETAIL:  streaming transactions committing after 0/18631E8, reading WAL from 0/18631B0
05:31:47  ip-10-0-2-127  LOG:  logical decoding found consistent point at 0/18631B0
DETAIL:  There are no running transactions.
05:31:47  ip-10-0-2-127  LOG:  logical decoding found consistent point at 0/18631E8
DETAIL:  Logical decoding will begin using saved snapshot.
05:31:47  ip-10-0-2-127  LOG:  exported logical decoding snapshot: "000002C6-1" with 0 transaction IDs
05:31:47  ip-10-0-2-127  LOG:  starting logical decoding for slot "bdr_12156_6347497366450917394_1_12156__"
DETAIL:  streaming transactions committing after 0/1863228, reading WAL from 0/18631E8
05:31:47  ip-10-0-2-127  LOG:  logical decoding found consistent point at 0/18631E8
DETAIL:  Logical decoding will begin using saved snapshot.
```

The following HAProxy config is generated.

```
ubuntu@ip-10-0-3-168:~$ cat /etc/haproxy/haproxy.cfg

global
    log /dev/log    local0
    log /dev/log    local1 notice
    chroot /var/lib/haproxy
    stats socket /run/haproxy/admin.sock mode 660 level admin
    stats timeout 30s
    user haproxy
    group haproxy
    daemon

    # Default SSL material locations
    ca-base /etc/ssl/certs
    crt-base /etc/ssl/private

    # Default ciphers to use on SSL-enabled listening sockets.
    # For more information, see ciphers(1SSL). This list is from:
    #  https://hynek.me/articles/hardening-your-web-servers-ssl-ciphers/
    ssl-default-bind-ciphers ECDH+AESGCM:DH+AESGCM:ECDH+AES256:DH+AES256:ECDH+AES128:DH+AES:ECDH+3DES:DH+3DES:RSA+AESGCM:RSA+AES:RSA+3DES:!aNULL:!MD5:!DSS
    ssl-default-bind-options no-sslv3

defaults
    log global
    mode    http
    option  httplog
    option  dontlognull
        timeout connect 5000
        timeout client  50000
        timeout server  50000
    errorfile 400 /etc/haproxy/errors/400.http
    errorfile 403 /etc/haproxy/errors/403.http
    errorfile 408 /etc/haproxy/errors/408.http
    errorfile 500 /etc/haproxy/errors/500.http
    errorfile 502 /etc/haproxy/errors/502.http
    errorfile 503 /etc/haproxy/errors/503.http
    errorfile 504 /etc/haproxy/errors/504.http



frontend tcp-10031685432
  bind 10.0.3.168:5432
  mode tcp
  default_backend tcp5432bdrdemo1

backend tcp5432bdrdemo1
  mode tcp
  server tcp5432bdrdemo1-100316810917 10.0.3.168:10917 maxconn 1024
  server tcp5432bdrdemo1-10014510917 10.0.1.45:10917 maxconn 1024
  server tcp5432bdrdemo1-100212710917 10.0.2.127:10917 maxconn 1024

```

**Review by huntc — APPROVED** [review]

**Pull request merged**